react/features/prejoin/components/web/Prejoin.tsx

```diff
@@ -393,11 +402,11 @@ const Prejoin = ({
                             displayName = { name }
                             participantId = { participantId }
                             size = { 72 } />
-                        <div className = { classes.avatarName }>{name}</div>
+                        {isDisplayNameVisible && <div className = { classes.avatarName }>{name}</div>}
                     </div>
                 )}
 
-                {showErrorOnJoin && <div
+                {showDisplayNameField && <div
```

Comment:
When is this going to be displayed now?

Reply:
Only when prejoingConfig.hideDisplayName is false, you don't have a display name and requireDisplayName is true.